debian/changelog

```diff
@@ -1,3 +1,38 @@
+adsys (0.12.0) mantic; urgency=medium
+  [ Denison Barbosa ]
+  [ Didier Roche ]
+  [ Gabriel Nagy ]
+  [ Jean-Baptiste Lallement ]
+  * Fix DCONF_PROFILE not considering default_domain_suffix on sssd.conf
+  * Go implementation for the user mount handler
+  * Remove Rust source code from adsys
+  * Rework Kerberos ticket handling logic
```

Comment:
@GabrielNagy, I feel like this isn't enough to say everything about what you did for the samba stuff, so what else should I add?